src/corehost/build.sh

```diff
@@ -19,18 +19,21 @@ init_rid_plat()
             echo "__rid_plat is $__rid_plat"
         fi
     else
+        __rid_plat=""
         if [ -e /etc/os-release ]; then
             source /etc/os-release
-
             if [[ "$ID" == "rhel" && $VERSION_ID = 7* ]]; then
                 export __rid_plat="rhel.7"
             elif [[ "$ID" == "centos" && "$VERSION_ID" = "7" ]]; then
                 export __rid_plat="rhel.7"
             else
                 export __rid_plat="$ID.$VERSION_ID"
             fi
-        else
-            export __rid_plat=
+        elif [ -e /etc/redhat-release ]; then
+            local redhatRelease=$(</etc/redhat-release)
+            if [[ $redhatRelease == "CentOS release 6."* || $redhatRelease == "Red Hat Enterprise Linux Server release 6."* ]]; then
+               __rid_plat="rhel.6"
```

Comment:
There is no need to export it, the export is needed only if we need to pass it to a child process, which we don't do. I have somehow missed the fact that the rest of the function uses the export, so I'll unify it.
.